### PR TITLE
Fixed Grammer in Readme

### PR DESCRIPTION
**Pull request by the-rebooted-coder**

### CODE REVIEW

**Comment by the-rebooted-coder** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ More information about building GoogleTest can be found at
 
 ## Features
 
-*   An [xUnit](https://en.wikipedia.org/wiki/XUnit) test framework.
+*   A [xUnit](https://en.wikipedia.org/wiki/XUnit) test framework.
```
Comment (derekmauro) [inline README.md:45]: I don't think this change is correct. "An" is used when a word begins with a vowel sound, and here is begins with an "ex" sound. Think "an xray".

File: README.md [hunk 2]
```diff
@@ -94,11 +94,9 @@ CMake is supported on a best-effort basis and by the community.
 In addition to many internal projects at Google, GoogleTest is also used by the
 following notable projects:
 
-*   The [Chromium projects](http://www.chromium.org/) (behind the Chrome browser
-    and Chrome OS).
+*   The [Chromium projects](http://www.chromium.org/) (behind the Chrome browser and Chrome OS).
```
Comment (derekmauro) [inline README.md:97]: This appears to be a whitespace only change. Please revert.

File: README.md [hunk 3]
```diff
@@ -94,11 +94,9 @@ CMake is supported on a best-effort basis and by the community.
...
 *   The [LLVM](http://llvm.org/) compiler.
-*   [Protocol Buffers](https://github.com/google/protobuf), Google's data
-    interchange format.
+*   [Protocol Buffers](https://github.com/google/protobuf), Google's data interchange format.
```
Comment (derekmauro) [inline README.md:99]: This appears to be a whitespace only change. Please revert.

**Review by derekmauro — CHANGES_REQUESTED** [review]

**Pull request closed without merging**